jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UISelectAssert.java

```diff
@@ -23,167 +23,220 @@
     }
+    
     @JDIAction("Assert that '{name}' is collapsed")
-    public A collapsed() {
-        jdiAssert(element.list().isHidden() ? "collapsed" : "expanded", Matchers.is("expanded"));
+    public A collapsed(String... messages) {
+        jdiAssert(element.list().isHidden() ? "collapsed" : "expanded", Matchers.is("expanded"), messages);
         return (A) this;
     }
+    
     @JDIAction("Assert that '{name}' is disappeared")
-    public A disappear() {
-        jdiAssert(element.isHidden() ? "hidden" : "displayed", Matchers.is("hidden"));
+    public A disappear(String... messages) {
+        jdiAssert(element.isHidden() ? "hidden" : "displayed", Matchers.is("hidden"), messages);
         return (A) this;
     }
-
+    
     @JDIAction("Assert that '{name}' is hidden")
-    public A hidden() {
-        return disappear();
+    public A hidden(String... messages) {
+        return disappear(messages);
     }
-    public A notAppear() {
-        return notAppear(TIMEOUT.get());
+    
+    public A notAppear(String... messages) {
+        return notAppear(TIMEOUT.get(), messages);
     }
-
+    
     @JDIAction(value = "Assert that '{name}' does not appear during {0} seconds", timeout = 0)
-    public A notAppear(int timeoutSec) {
+    public A notAppear(int timeoutSec, String... messages) {
         boolean result = new Timer(timeoutSec * 1000)
                 .wait(() -> element.isDisplayed());
-        jdiAssert(result ? "displayed" : "hidden", Matchers.is("hidden"));
+        jdiAssert(result ? "displayed" : "hidden", Matchers.is("hidden"), messages);
         return (A) this;
     }
-
-    public Compare exact(int count) {
-        return new Compare(count, this, true);
+    
+    public Compare exact(int count, String... messages) {
+        return new Compare(count, this, true, messages);
     }
-    public Compare atLeast(int count) {
-        return new Compare(count, this, false);
+    
+    public Compare atLeast(int count, String... messages) {
+        return new Compare(count, this, false, messages);
     }
-    public Compare no() {
-        return exact(0);
+    
+    public Compare no(String... messages) {
+        return exact(0, messages);
     }
-    public Compare all() {
-        return exact(element.size());
+    
+    public Compare all(String... messages) {
+        return exact(element.size(), messages);
     }
-    public Compare onlyOne() {
-        return exact(1);
+    
+    public Compare onlyOne(String... messages) {
+        return exact(1, messages);
     }
-
+    
     public class Compare implements JAssert {
-
+        
         public int count;
         public String name;
         public String type;
-        UISelectAssert<A,E> dtAssert;
+        UISelectAssert<A, E> dtAssert;
         boolean exact;
-        public JDIBase base() { return UISelectAssert.this.base(); }
-
-        private Compare(int count, UISelectAssert<A,E> dtAssert, boolean exact) {
+        String[] messages;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)